app/workers/import_ml_data_worker.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'csv'
+# Worker to bulk import Machine Learnt (ML) extracts from web accessibly located csv
+class ImportMLDataWorker
+  include Sidekiq::Worker
+  sidekiq_options retry: 2
+
+  def perform(csv_filepath, workflow_id)
+    workflow = Workflow.find(workflow_id)
+    upsert_extracts csv_filepath, workflow
+    run_workflow_reducers(workflow) if workflow&.reducers&.any?
+    project = Project.find(workflow.project_id)
```

Comment:
Will this raise an error if the project isn't saved in the DB? 

Caesar traditionally processes data for either workflows or projects in isolation. When creating a workflow it won't necessarily create the linked Project resource so these lookups may raise.